From: Ronny Chevalier <chevalier.ronny@gmail.com>
Date: Sat, 21 Jun 2014 22:07:09 +0200
Subject: build-sys: add -pthread flag for libsystemd-shared

src/shared/async.c uses pthread so it will fail at link time if we link
only to libsystemd-shared and use async

(cherry picked from commit 8e75477abdd838d3beddc5fd1c6a7707b22748b6)
---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 75d62f0..71fc911 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -823,7 +823,8 @@ nodist_libsystemd_shared_la_SOURCES = \
 
 libsystemd_shared_la_CFLAGS = \
 	$(AM_CFLAGS) \
-	$(SECCOMP_CFLAGS)
+	$(SECCOMP_CFLAGS) \
+	-pthread
 
 # ------------------------------------------------------------------------------
 noinst_LTLIBRARIES += \
